bin/node/cli/src/chain_spec.rs

```diff
@@ -393,6 +393,7 @@ fn pangolin_build_spec_genesis() -> pangolin_runtime::GenesisConfig {
 		},
 		darwinia_evm: pangolin_runtime::EVMConfig { accounts: evm_accounts },
 		dvm_ethereum: Default::default(),
+		darwinia_bridge_ethereum_bsc: Default::default(),
```

Comment:
What genesis header should be used here?

Reply:
https://bscscan.com/

For Pangolin, maybe we should use its testnet: https://testnet.bscscan.com/

@songtianyi Please select and use a recent epoc/header as the genesis data.